user_nl_cam_0001

```diff
@@ -6,3 +6,4 @@
  empty_htapes  = .true. 
  fincl1        = 'PHIS:I' 
  nhtfrq        = -6 
+fincl2        = 'SST:I'
```

Comment:
Are we adding a second history file ... doesn't nhtfrq need a second value then?